src/server/shared/SharedDefines.h

```diff
@@ -458,7 +458,7 @@ enum SpellAttr1 : uint32
     SPELL_ATTR1_DISPEL_AURAS_ON_IMMUNITY         = 0x00008000, // TITLE Immunity cancels preapplied auras DESCRIPTION For immunity spells, cancel all auras that this spell would make you immune to when the spell is applied
     SPELL_ATTR1_UNAFFECTED_BY_SCHOOL_IMMUNE      = 0x00010000, // TITLE Unaffected by school immunities DESCRIPTION Will not pierce Divine Shield, Ice Block and other full invulnerabilities
     SPELL_ATTR1_UNAUTOCASTABLE_BY_PET            = 0x00020000, // TITLE Cannot be autocast by pet
-    SPELL_ATTR1_UNK18                            = 0x00040000, // TITLE Unknown attribute 18@Attr1 DESCRIPTION Stun, Polymorph, Daze, Hex - CC?
+    SPELL_ATTR1_PREVENTS_ANIM                    = 0x00040000, // TITLE NYI, auras applies UNIT_FLAG_PREVENT_EMOTES_FROM_CHAT_TEXT
```

Comment:
```suggestion
    SPELL_ATTR1_PREVENTS_ANIM                    = 0x00040000, // TITLE NYI, aura applies UNIT_FLAG_PREVENT_EMOTES_FROM_CHAT_TEXT
```

Reply:
Good point, it should be either "aura applies" or "auras apply", depending on singular or plural aura.